### DISCUSSION TITLE
Cannot view any repository: „Something went wrong. Try again“

### DISCUSSION THREAD

**Discussion by phNis** [community/community#69637] [Mobile] [answered]
Mobile: Clicking any repository (my own, featured, whatever) results in the screen "Something went wrong." with a "Try again" button. Clicking that button does not change the outcome. Deleting & reinstalling the app did not help. 
GitHub Desktop: selectively shows commits to own repository; some are shown, others not. No change on re-fetching. 
GitHub.com: works as expected. 

<details>

<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.133.1 (112099449)
Device: iPhone 11 Pro (iOS 17.0.2)
</details>

**Comment by elisealix22** [comment]
Hi @phNis , we recently added more detailed error information in v1.132 and greater. Is there any chance you can upgrade and see if you can get any additional hints as to what could be going wrong? I'm sorry you're running into trouble!

**Reply by phNis** [reply, marked as answer]
Hey Elise,
I just updated to 1.134.2, but no change (also no additional error information). 

But looking around the app to see what worked and what didn't, I figured I'd sign out of my profile and see if that fixed things - and it did. After signing back in, I can access my own and other repositories fine. It just never occurred to me to try because there's no indication there is anything wrong with the sign-in - I can post from here, and the app even gets used as second factor for web logins. 
So, long story short: "Have you tried turning it off and on again?"

I will check GitHub Desktop later. 

Thanks for getting back to me on this, though!

**Reply by elisealix22** [reply]
I'm glad you were able to resolve this but it's still not great that you were able to get in that state! We will definitely keep an eye on this as permissions are tricky when there are multiple organizations involved.
